plugins/mcreator-localization/lang/texts.properties

```diff
@@ -3211,11 +3217,15 @@ common.close=Close
 common.cancel=Cancel
 common.confirmation=Confirmation
 common.mod_element_name=Mod element name
-common.not_applicable=N/A
+common.namespace=Namespace
 common.name_already_exists=This name already exists
+common.not_applicable=N/A
 common.warning=Warning
 components.collapsible_panel.expand=Click to expand
 components.collapsible_panel.collapse=Click to collapse
+components.string_list.item=Entry at index {0}
```

Comment:
Are we sure those are still used?

Reply:
Forgot about this one. Buttons do still have their tooltips, however.

Reply:
✔️ Fixed